tests/resources/configs/rule_bindings/team-7-rule-bindings.yml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+rule_bindings:
+  T12_URI_NATIVE_BQ_NOT_NULL:
+    entity_uri: bigquery://locations/<my-gcp-dataplex-region-id>/lakes/<my-gcp-dataplex-lake-id>/zones/<my-gcp-dataplex-zone-id>/projects/<my-gcp-project-id>/datasets/austin_311/tables/311_service_requests
```

Comment:
I wonder if we should use a public dataset or something created and owned by us.

- Theoretically, public dataset can change (? for example, adding a new column), which can then make our test fail. I don't think we should create such dependency for our tests.
- Public dataset does not provide full coverage of our test scenarios. For example, `partition_fields` is either `None` or `null` for all the testing cases in this PR. I think we should have at least a test case where it's not `None` or `null`.

Reply:
1. Theoretically, public dataset can change (? for example, a new column added), which can then make our test fail. I don't think we should create such dependency for our tests.

> I have made a copy of public dataset in our project, so the table structure is not going to change until we make a change. So our tests will run independent of public dataset.

2.  Public dataset does not provide full coverage of our test scenarios. For example, partition_fields is either None or null for all the testing cases in this PR. I think we should have at least a test case where it's not None or null.

>  Yes, I agree I will add a test for the same.